R/backend-postgres.R

```diff
@@ -250,6 +250,34 @@ sql_query_explain.PqConnection <- function(con, sql, format = "text", ...) {
 #' @export
 sql_query_explain.PostgreSQL <- sql_query_explain.PqConnection
 
+#' @export
+sql_query_insert.PqConnection <- function(con, x_name, y, by,
```

Comment:
Do you mind taking a quick look over the other `sql_query_insert` methods and see if there are any opportunities to reuse code?